modules/nw-mutual-tls-auth.adoc

```diff
@@ -61,3 +61,14 @@ $ oc edit IngressController default -n openshift-ingress-operator
       allowedSubjectPatterns:
       - "^/CN=example.com/ST=NC/C=US/O=Security/OU=OpenShift$"
 ----
++
+[NOTE]
```

Comment:
I was looking at the bug and seems like if the user doesn't do this, they could get a 403 error, if I'm understanding the bug correctly. In that case, I wonder if this information should be a step, maybe even an optional step, instead of a NOTE?

Reply:
I had same thought earlier . just wanted confirmation.

Reply:
@skrthomas can you check now.

Reply:
/lgtm 

Sorry I missed this in my inbox!!!